src/libm/sleefsimdsp.c

```diff
@@ -1712,6 +1752,27 @@ EXPORT CONST vfloat xfmodf(vfloat x, vfloat y) {
   return ret;
 }
 
+EXPORT CONST vfloat xsinpif_u05(vfloat d) {
```

Comment:
These are required by the dispatcher.
Anyway, I am currently implementing these functions and I expect it will finish soon.